### PR TITLE
Use Signals slots new syntax

### PR DESCRIPTION
**Pull request by ferranlala**

### CODE REVIEW

File: src/vinylcontrol/vinylcontrolmanager.cpp [hunk 1]
```diff
@@ -33,8 +33,10 @@ VinylControlManager::VinylControlManager(QObject* pParent,
             AudioInput(AudioInput::VINYLCONTROL, 0, 2, i), m_pProcessor);
     }
 
-    connect(&m_vinylControlEnabledMapper, SIGNAL(mapped(int)),
-            this, SLOT(slotVinylControlEnabledChanged(int)));
+    connect(&m_vinylControlEnabledMapper,
+            qOverload<int>(&QSignalMapper::mapped),
```
Comment (daschuer) [inline src/vinylcontrol/vinylcontrolmanager.cpp]: #include "util/compatibility.h" is missing

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you for continuing to taking care about this.

File: src/vinylcontrol/vinylcontrolmanager.cpp (hunk 1 above)
Comment (ferranpujolcamins) [reply src/vinylcontrol/vinylcontrolmanager.cpp]: It's already in the file

Comment (ferranpujolcamins) [reply src/vinylcontrol/vinylcontrolmanager.cpp]: Furthermore, is it needed to use qOverload now that we target c++14?

According to [the docs](https://doc.qt.io/qt-5/qtglobal.html#qOverload) we need qt >= 5.7 and c++14, wich we have.

Comment (daschuer) [reply src/vinylcontrol/vinylcontrolmanager.cpp]: Ah, got it. "util/compatibility.h" does not contain the c++14 version. 
So we need either add it or use QOverload<int>::of() instead. 

For Ubuntu Xenial we need QT 5.5 compatibility.

Comment (ferranpujolcamins) [reply src/vinylcontrol/vinylcontrolmanager.cpp]: Fixed. 

My understanding was that we only supported latest LTS, Bionic with Qt 5.9.

Comment (daschuer) [reply src/vinylcontrol/vinylcontrolmanager.cpp]: We have discussed that, but the final conclusion was to support Xenial until there is a strong reasons to drop it. 

Now it builds again. Thank you.

**Pull request merged**